.travis.yml

```diff
@@ -14,6 +14,8 @@ env:
     - NPROC_MAX=8 BUILDTEST_MCU_GROUP=cortex_m4
 
 before_install:
+    - source ./dist/tools/pr_check/check_labels.sh
+    - test "$BUILDTEST_MCU_GROUP" = "static-tests" || check_gh_label "Ready for CI build" || exit 1
```

Comment:
Add a `test -z "$TRAVIS_PULL_REQUEST" || ` to the beginning to not skip everything if the build is not a PR build, e.g. building a commit on a pushed branch.
I'm using the same kind of test on Drone to be able to build all branches that I push to my own Github repo.
This also affects the label check because there are no labels to check if there is no PR.